### PR TITLE
Update GH integration permissions docs

### PR DESCRIPTION
**Pull request by karensawrey**
Reverse the order of integration instructions & add a link to the list of possible filenames Buildkite will be looking for.

### CODE REVIEW

**Comment by karensawrey** [comment]
@plaindocs I'll test the whole process anew, yep. Because my results with an existing GH account are such that only minor changes to docs are necessary, but for a new user it might be a whole different kettle of fish.

**Comment by karensawrey** [comment]
@plaindocs I removed the unnecessary stuff, reordered ti PR a bit, but it could use more rework, I think. Please take a look at it.

File: pages/integrations/github.md.erb [hunk 1]
```diff
@@ -159,3 +145,15 @@ For example, if you have a monorepo containing three applications, you could use
 ## Build skipping
 
 <%= render_markdown partial: 'integrations/build_skipping' %>
+
+## Connect your Buildkite account to GitHub using OAuth
+
+To connect your GitHub account:
+
+1. Open Buildkite's _Personal Settings_.
```
Comment (plaindocs) [inline pages/integrations/github.md.erb:153]:
```suggestion
1. Open your [Buildkite _Personal Settings_](https://buildkite.com/user/settings).
```

**Review by plaindocs — APPROVED** [review]
Looks good to me.

**Pull request merged**